cmd/shell.go

```diff
@@ -335,3 +364,15 @@ func LoadCert(path string) (cert []byte, err error) {
 	cert = block.Bytes
 	return
 }
+
+func DebugServer(addr string) {
+	if addr == "" {
+		log.Fatalf("unable to boot debug server because no address was given for it. Set debugAddr.")
```

Comment:
Is this really a fatal error, or just a situation where we chose not to enable debugging?